<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialDecisionDockWidgetBase</class>
 <widget class="QDockWidget" name="SpatialDecisionDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>530</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>PASCAL</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current Scenario:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QTabWidget" name="sampleWidgets">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <item>
         <widget class="QComboBox" name="selectNetworkCombo">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selectNodeCombo"/>
        </item>
        <item>
         <widget class="QPushButton" name="loadAmsterdamNoordButton">
          <property name="text">
           <string>Load data Amsterdam Noord</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analysisTab">
       <attribute name="title">
        <string>analysis</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="2" column="1">
         <widget class="QPushButton" name="stationDistanceButton">
          <property name="text">
           <string>Station Distance</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="stationDistanceSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="sliderValue">
          <property name="text">
           <string>500</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="createScenarioButton">
          <property name="text">
           <string>Create Scenario</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Myriad Pro</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Here you can place new nodes</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>1.</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="selectTransportCombo">
          <item>
           <property name="text">
            <string>Select mode of transport</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tram</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>metro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ferry</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QCheckBox" name="visibilityCheckBox">
          <property name="text">
           <string>Show me all nodes and road types</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>2.</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="stopNameEdit">
          <property name="text">
           <string>&lt;Enter name of node&gt;</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>3.</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QPushButton" name="addNodesButton">
          <property name="text">
           <string>Add Node</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>visualisation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="visualisationTab">
       <attribute name="title">
        <string>reporting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="statistics1Table"/>
        </item>
        <item>
         <widget class="QComboBox" name="scenario1Box"/>
        </item>
        <item>
         <widget class="QTableWidget" name="statistics2Table"/>
        </item>
        <item>
         <widget class="QComboBox" name="scenario2Box"/>
        </item>
        <item>
         <widget class="QPushButton" name="statisticsButton">
          <property name="text">
           <string>Statistics</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/pascal.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="scenarioCombo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
